openapi.yaml

```diff
@@ -0,0 +1,154 @@
+openapi: 3.0.0
+info: 
+  title: To-do API
+  description: To-do app api
+  version: '1.0'
+  
+paths: 
+  /to-dos:
+    get:
+      summary: Return list of to-do
+      responses:
+        '200':
+          description: A Json array of to-dos
+          content: 
+            application/json:
+              schema:
+                type: array
```

Comment:
bu şekilde path altında array oluşturmaktansa schema içerisinde array'ı oluşturup buraya direk o schemayı verebilirsin örneğin '#/components/schemas/ToDoBundleResponse